content/output/demographic/_index.Rmd

```diff
@@ -0,0 +1,98 @@
+---
+title: "Create Demographic table"
+author: "Tadeusz Lewandowski"
+date: "16/08/2022"
+output: html_document
+description: "An example of how to generate an Demogrphic output based on ADaM dataset"
+---
+
+## Introduction
+
+This guide will show you how two pharmaverse packages, along with some from tidyverse, can be used to create a Demographic table as an end-to-end, using the Admiral ADSL data as input.
+
+The two packages used with a brief description of their purpose are as follows:
+
+* [`{rtables}`](https://roche.github.io/rtables/index.html): designed to create and display complex tables with R.
```

Comment:
Should we mention here some sentence like:
Alternative tabulation packages exist that could be considered here instead such as `{Tplyr}` (see https://pharmaverse.org/e2eclinical/tlg/)

Reply:
I will have a separate example for Tplyr with the ad-hoc module. For simplicity of the code, I aimed to present the same static output also in the Shiny using the teal.module.clinical.